dapp-developer-guide/getting-started.md

```diff
@@ -0,0 +1,275 @@
+### Meaning of ENS
+
+The Ethereum Name Service (ENS) is a useful tool for dapp developers. ENS is like DNS (Domain Name System), in that it maps a memorable shortcut to an address. Using ENS we can map the friendly name ethereum.eth to the rather unfriendly 0xfB6916095ca1df60bB79Ce92cE3Ea74c37c5d359. Subsequently, we can use the friendly name in-place of the address, making it easier to remember, and reducing the chance of errors.<br/>
+An ENS name has a root name ethereum.eth which can contain sub-names like wallet.ethereum.eth. Whoever successfully bid for the root name in a Vickrey auction owns it.<br/>
+ENS is the Ethereum Name Service, a distributed, open, and extensible naming system based on the Ethereum blockchain.
+
+### Why Developers need to use the ENS
+ENS doesn't just map to addresses. ENS is highly extensible and supports many types of mappings. It is possible to map the same friendly name to multiple endpoints at the same time. <br/>
+ENS gets rid of the necessity of copying or typing lengthy hexadecimal addresses. With ENS, you’ll be able to send funds to the domain name instead of the long address, engage with your smart contract at mycontract.eth, and check out swarm-hosted sites at swarmsite.eth. <br/>
+
+ENS is a completely decentralized system. Creating new domains under the “.eth” top-level domain is possible by an auction process that takes place on the Ethereum blockchain, and anyone can procure a domain for themselves by taking part.
+
+For example the ethereum.eth name could point to:
+- A multisig wallet contract address
+- Public encryption keys for secure communication
+- Website content (via IPFS hash or multihash)
+
+It could be described with a phone-book metaphor, top-level domains, like ‘.eth’ and ‘.test’ are owned by smart contracts called registrars, which specify rules governing the allocation of their subdomains. Anyone may, by following the rules imposed by these registrar contracts, obtain ownership of a domain for their own use.
+Because of the hierarchical nature of ENS, anyone who owns a domain at any level may configure subdomains - for themselves or others - as desired. For instance, if Alice owns 'alice.eth', she can create 'pay.alice.eth' and configure it as she wishes.
+
+
+#### ENS Subdomains
+They can also give more of an organizational feel to users, where the domain represents the organization and the subdomain resembles the structure of email addresses. A novel use case currently being implemented by Tenzorum Project is to use your subdomain as a username/login to Ethereum ĐApps. <br/>
+Subdomains map names to addresses exactly the same way as domains do, but you don’t need to go through an auction process to create one. Luckily, there is no minimum length limit on ENS subdomains, either. To register a subdomain, you have to be the owner of the domain as a prerequisite.
```

Comment:
>To register a subdomain, you have to be the owner of the domain as a prerequisite.
You can also register subdomains of other ENS Names that you don't own
but that their owners have allowed others to create subdomains of, for free in some cases, and at a periodic/annual cost in other cases.

These subdomains will soon be registerable through the ENS Subdomain Registrar